src/face.rs

```diff
@@ -0,0 +1,8 @@
+use serde::{Serialize, Deserialize};
+
+#[derive(Serialize, Deserialize, Debug)]
+pub enum Expression {
+    Unknown,
```

Comment:
I dont really see a need for an "Unknown" expression type - the models will only answer a yes/no question for each expression (e.g. "is the person in this photo angry").